main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -38,6 +39,21 @@ object Flix {
     * The reserved Flix delimiter.
     */
   val Delimiter: String = "$"
+
+  /**
+    * The file extension for intermediate representation files.
+    */
+  val IrFileExtension = "flixir"
+
+  /**
+    * The maximum width of the intermediate representation files.
+    */
+  val IrFileWidth = 80
+
+  /**
+    * The number of spaces per indentation in the intermediate representation files.
+    */
+  val IrFileIndentation = 4
```

Comment:
@magnus-madsen I don't know if this is the right place for these constants, but I also think it's the easiest place to find.

Reply:
👍